libs/tmelcrypt/Cargo.toml

```diff
@@ -18,4 +18,4 @@ serde-big-array = "0.3.0"
 
 [dependencies.ed25519-dalek]
 features = ["legacy_compatibility"]
-version = "1.0.0-pre.3"
+version = "=1.0.0-pre.3"
```

Comment:
@nullchinchilla had to add this because for some reason certain dependencies would go 1.0.0-pre.3 or greater if I didn't pin it here.

Reply:
we'll probably just use the later version anyway. When I first wrote tmelcrypt 1.0.0 wasn't released yet but now it is.